docs/gallery_scripts_template/plot_peakmap_marginals.py

```
@@ -6,17 +6,24 @@
 A chromatogram is shown along the x-axis and a spectrum is shown along the y-axis.
 """
 
+pd.options.plotting.backend = "TEMPLATE"
```

Ruff (0.14.8)

Undefined name `pd`

(F821)